apps/src/code-studio/pd/workshop_dashboard/components/survey_results/choice_responses.jsx

```diff
@@ -103,13 +104,14 @@ export default class ChoiceResponses extends React.Component {
         0
       );
     });
+    const {facilitators} = this.props;
 
     const headerRow = (
       <tr key="header">
         <td />
         {facilitatorNames.map((name, i) => (
           <td colSpan={2} style={{paddingLeft: '20px'}} key={i}>
-            {name}
+            {facilitators && facilitators[name] ? facilitators[name] : name}
```

Comment:
If it's going to be a one-liner anyway, this is a little more compact:
```suggestion
            {facilitators && facilitators[name] || name}
```